packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -474,6 +474,9 @@ export class Cluster extends Resource implements ICluster {
   }
 
   private configureAutoScalingGroup(autoScalingGroup: autoscaling.AutoScalingGroup, options: AddAutoScalingGroupCapacityOptions = {}) {
+    if (autoScalingGroup.connections?.securityGroups) {
+      this.connections.connections.addSecurityGroup(...autoScalingGroup.connections.securityGroups);
+    }
```

Comment:
I ask because I just want to make sure that there is a tangible change to at least one integ test template generated by this line of code. If so, then we're all good and I'll approve; if not, we need to make sure we can figure out what change we expect to be applied to the clsuter security groups or autoscaling group connections.

Reply:
These changes are tested over here my modifying this integration test and this will modify snapshot to add security groups from ASG connections to Cluster connections.
https://github.com/aws/aws-cdk/pull/25699/files#diff-05228637eeb8241609924d2fc7acce96e8aa522e1aafe36ca7d5a47c3b73ab16R53